### ISSUE TITLE
[BUG]Running some unadapted applications causes WSA to crash

### ISSUE THREAD

**Issue by YanYiGe2023** [MustardChef/WSABuilds#452]
### Describe the bug

I've tried to run the game Teal Archive on wsa, and if I set vulkan to system default or dx12, a few seconds after opening the game Teal Archive, the wsa crashes completely with "Windows Subsystem for Android has stopped working"
If I set Vulkan to none, the game doesn't work properly, the screen will stay black after the icon, and the vmmemwsa process CPU usage is around 3%, which proves that the app is not running at all
It's worth mentioning that in Google Play the Teal Profile will show up as "This app doesn't work well with your tablet", but Teal Profile is not an isolated case, and in fact many games don't work

### Steps to reproduce the issue

As above

### Expected behaviour

Runs normally, doesn't crash

### Downloaded Build Of WSA

Tried many versions,
1、WSA_2306.40000.4.0_x64_Release-Nightly-MindTheGapps-13.0-RemovedAmazon_Windows_10Nothing happens
2、WSA_2309.40000.2.0_x64_Release-Nightly-with-magisk-26.3.26300.-stable-MindTheGapps-13.0_Windows_10Stuck in the boot icon
3、WSA_2311.40000.4.0_x64_Release-Nightly-MindTheGapps-13.0-RemovedAmazon_Windows_10 Consistent with the situation above
4、WSA_2407.40000.0.0_x64（Windows11）Consistent with the situation above

### Windows build number

1.2.3 is Windows10 22h2
4 Windows11 23h2

### PC Specification

1.2.3、i5-10400 8GB NoDiscrete GPU
4、AMD Ryzen 7 4800H with Radeon Graphics 16GB

**Comment by Schefflin** [comment]
yes, do not install those apps play store doesn't recommend. most likely reason is those apps don't be supported to run on "x86" or "x86_64", otherwise WSA vulkan version is too old.
